roles/imageset_mirroring/README.md

```diff
@@ -0,0 +1,186 @@
+# imageset_mirroring
+
+Mirror OpenShift operator catalogs and images using oc-mirror.
+
+## Description
+
+This role provides three different approaches to mirror operators from a source catalog to a target registry:
+
+- **mirror-latest**: Mirror only the latest version of specified operators
+- **mirror-all**: Mirror all channels and versions of specified operators
+- **mirror-custom**: Mirror using a pre-defined custom ImageSetConfiguration file
+
+## Requirements
+
+- `oc-mirror` binary (automatically downloaded by the role)
+- Network access to source and target registries
+- Valid authentication for registries (if required)
+
+## Role Variables
+
+### Common Variables
+
+| Variable | Type | Required | Default | Description |
+|----------|------|----------|---------|-------------|
+| `im_mirror_target` | string | yes | - | Target registry for mirroring (e.g., `registry.example.com:5000`) |
+| `im_allow_insecure_registries` | boolean | no | `true` | Allow insecure registries (sets `--dest-tls-verify=false` and `--src-tls-verify=false`) |
+| `im_keep_working_dir` | boolean | no | `false` | Keep temporary working directory after mirroring completes |
+| `im_auths_file` | string | no | - | Path to registry authentication file (e.g., `/path/to/pull-secret.json`) |
+
+### Task-Specific Variables
+
+#### mirror-latest & mirror-all
+
+| Variable | Type | Required | Description |
+|----------|------|----------|-------------|
+| `im_source_index` | string | yes | Source catalog index (e.g., `registry.redhat.io/redhat/redhat-operator-index:v4.20`) |
+| `im_mirror_list` | dictionary | yes | Dictionary of operators to mirror (keys are operator names) |
```

Comment:
What about renaming `im_mirror_list` to: `im_operators` ? this one named as list but being a dictionary could lead to misunderstandings